### ISSUE TITLE
memory leak in "readImageEmscriptenFSDICOMFileSeries" method

### ISSUE THREAD

**Issue by vahidbahramian** [InsightSoftwareConsortium/itk-wasm#421]
I am trying to read a DICOM series in my node js server but when I call the readImageEmscriptenFSDICOMFileSeries method memory starts to grow up and never come back and it leads to crash after a while.

```
readImageLocalDICOMFileSeries("D:\\misSolution_2010\\misSoloution\\MedicalImageManagement_Application\\TempFile").then((image) => {
        client.emit('response', image);
      })
      .catch((error) => {
        console.log(error);
        client.emit('response', null);
      })
```

**Comment by thewtex** [comment]
@vahidbahramian is it possible to create a small test script that reproduces the memory leak?

**Comment by thewtex** [comment]
This may be related to the use of Embind and should be addressed in a future release.

**Comment by thewtex** [comment]
Addressed in `1.0.0.a.10`
